apis/python/test/common.py

```diff
@@ -363,3 +364,31 @@ def quantize_embeddings_int8(
     starts = ranges[0, :]
     steps = (ranges[1, :] - ranges[0, :]) / 255
     return ((embeddings - starts) / steps - 128).astype(np.int8)
+
+
+def setUpCloudToken():
```

Comment:
(Minor)(Could be a future PR) Would be nice to update `test_cloud.py` to also call this and `create_cloud_uri()`.

Reply:
Good suggestion, I will follow up to do this in a future PR